packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -956,3 +956,91 @@ reg:
     expires: never
     data_sensitivity:
       - interaction
+
+cad_firefox:
+  view:
+    type: event
+    description: |
+      User viewed the "Download Firefox for mobile" screen after choosing and submitting the "I don't have Firefox for mobile" option
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9607
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+  choice_view:
+    type: event
+    description: |
+      User viewed the "Connect another device" screen with choice options to download FF for mobile or not
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9607
+    data_reviews:
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+  choice_engage:
+    type: event
```

Comment:
This and `choice_submit` below need an `extra_keys` for 'reason'.

Reply:
Ah thanks, looking now. I see `ad_firefox.choice_engage: Invalid key index: reason` testing this locally now.